sdk/tests/deqp/functional/gles3/es3fOcclusionQueryTests.js

```diff
@@ -283,20 +297,34 @@ es3fOcclusionQueryTests.OcclusionQueryCase.prototype.iterate = function() {
     }
 
     gl.endQuery(this.m_queryMode);
-    gl.finish();
     gl.disable(gl.SCISSOR_TEST);
     gl.disable(gl.STENCIL_TEST);
     gl.disable(gl.DEPTH_TEST);
+    this.m_state = es3fOcclusionQueryTests.State.VERIFY;
+};
 
+es3fOcclusionQueryTests.OcclusionQueryCase.prototype.verify = function() {
     // Check that query result is available.
-    var resultAvailable = /** @type {boolean} */ (gl.getQueryParameter(query, gl.QUERY_RESULT_AVAILABLE));
-    assertMsgOptions(resultAvailable, 'Occlusion query failed to return a result after glFinish()', false, true);
+    var resultAvailable = /** @type {boolean} */ (gl.getQueryParameter(this.m_query, gl.QUERY_RESULT_AVAILABLE));
```

Comment:
In the core ES 3.0 spec those are the two query types. The EXT_disjoint_timer_query extension defines another one which will be affected by this change too (they're not used by the dEQP tests, though).